Week3/homework/js-exercises/shopping.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+let shoppingList = ['Appel', 'pear']
+
+function addToShoppingCart(oneMore){
+    shoppingList.push(oneMore)
+    if(shoppingList.length <=3 ){
+        console.log(shoppingList)
+        return `You bought ${shoppingList}!`}
```

Comment:
Don't forget to check the `formatting`. Here the `}` is not in a logical place, but this is for next time:)